src/translations/en.json

```
@@ -5691,8 +5691,8 @@
           },
           "move_card": {
             "header": "Choose a view to move the card to",
-            "error_title": "Impossible to move the card",
-            "error_text_section": "Moving a card to a section view is not supported yet. Use copy/cut/paste instead."
+            "strategy_error_title": "Impossible to move the card",
+            "strategy_error_text_strategy": "Moving a card to a strategy view is not supported."
```

```suggestion
            "strategy_error_text_strategy": "Moving a card to a auto generated view is not supported."
```

@piitaya what do you think? I don't like the sentence with *strategy view*, seems maybe confusing for the user

---

I agree. BTW, this message should never be displayed as it's not possible to select a strategy view when moving a card. It's only additional security.